services/app/apps/codebattle/assets/js/widgets/containers/UserProfile.jsx

```diff
@@ -309,18 +309,19 @@ const UserProfile = () => {
   return (
     <div className="container-lg">
       <div className="row">
-        <div className="col-12 col-md-3 my-4 cb-user-data">
-          <div className="mb-4 d-flex justify-content-center">
+        <div className="col-12 col-md-3 my-4 cb-user-data flex-column">
```

Comment:
```suggestion
        <div className="col-12 col-md-3 my-4 cb-user-data flex-column">
```
Да, я уже изменил направление флекса в этом пр. Хорошо, тогда я сейчас уберу кастомные кастомные стили и добавлю бутстрап стили.